src/argparse.c

```diff
@@ -133,6 +133,55 @@ static inline bool file_exists(const char *const path)
     return -1 != stat(path, &buf);
 }
 
+static inline void load_env(neu_cli_args_t *args, char **log_level_out,
+                            char **config_dir_out, char **plugin_dir_out)
+{
+
+    char *daemon = getenv(NEU_ENV_DAEMON);
+    if (daemon != NULL) {
+        if (strcmp(daemon, "1") == 0) {
```

Comment:
Question: is checking `NEURON_DAEMON` enough so that the following cases work ?
```
NEURON_DAEMON=1 ./neuron
NEURON_DAEMON=true ./neuron
```

Reply:
It is recommended not to provide too many options, as the parameters here will also be reflected in the configuration file.